spring-batch-notion/pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.3.5</version>
+    <relativePath/>
+  </parent>
+
+  <groupId>org.springframework.batch.extensions</groupId>
+  <artifactId>spring-batch-notion</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+
+  <name>Spring Batch Notion</name>
+  <description>Spring Batch extension for Notion</description>
+  <url>https://github.com/spring-projects/spring-batch-extensions/spring-batch-notion</url>
+  <inceptionYear>2023</inceptionYear>
+  <licenses>
+    <license>
+      <name>Apache 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>scordio</id>
+      <name>Stefano Cordio</name>
+      <email>stefano.cordio@gmail.com</email>
+      <url>https://github.com/scordio</url>
+    </developer>
+  </developers>
+
+  <scm>
+    <url>https://github.com/spring-projects/spring-batch-extensions/spring-batch-notion</url>
+  </scm>
+
+  <properties>
+    <java.version>17</java.version>
+    <notion-sdk-jvm.version>1.11.1</notion-sdk-jvm.version>
+  </properties>
+
+  <dependencies>
+    <!-- Compile -->
+    <dependency>
+      <groupId>com.github.seratch</groupId>
+      <artifactId>notion-sdk-jvm-core</artifactId>
+      <version>${notion-sdk-jvm.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.seratch</groupId>
+      <artifactId>notion-sdk-jvm-httpclient</artifactId>
+      <version>${notion-sdk-jvm.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.seratch</groupId>
+      <artifactId>notion-sdk-jvm-slf4j2</artifactId>
+      <version>${notion-sdk-jvm.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-beans</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.batch</groupId>
+      <artifactId>spring-batch-infrastructure</artifactId>
+    </dependency>
+    <!-- Test -->
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.tngtech.archunit</groupId>
+      <artifactId>archunit-junit5</artifactId>
+      <version>1.3.0</version>
```

Comment:
You have already a `<properties></properties>` section, so I would extract all these versions there to be centralized.

Reply:
I usually use properties only when they affect more than one dependency (e.g., `notion-sdk-jvm` in this specific case), and there is no BOM for them. Otherwise, I keep the versions in each dependency declaration.

I'm totally fine with moving all of them to the `properties` section. I take care of it now.

Reply:
Use whatever approach you are used to and confortable with. You can keep properties as they are now.